src/utils.ts

```diff
@@ -59,6 +61,10 @@ export function guid(length = 8) {
     return result;
 }
 
+export function getBufferHash(buf: Buffer) {
+    return createHash('sha256').update(buf).digest('hex');
```

Comment:
Should we allow setting the `algorithm` via the `options`?

Reply:
I don't think it's necessary. It is completely internal feature which is needed for calculating unique string based on file contents so it can use any algorithm and it will not affect the result.